### PR TITLE
Add Awesome List + Llama Index

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
I would like us to add a list of Awesome Panel Chat Resources by the community. It will help users understand to understand how powerful it is and how to get started.

I would also like to add a section on Llama Index. For Llama Index I would suggest we keep the code on Llama Hub. The [Llama Hub people have already agreed](https://github.com/run-llama/llama-hub/pull/703#discussion_r1416624184) that the upcoming `panel_chatbot` pack can grow into a multipage app of examples.

### CODE REVIEW

**Review by jerryjliu — COMMENTED** [review]
small nit: it is technically *LlamaIndex but I don't think that should block

**Review by jerryjliu — APPROVED** [review]

**Comment by MarcSkovMadsen** [comment]
> small nit: it is technically *LlamaIndex but I don't think that should block

Should be fixed now.

**Comment by MarcSkovMadsen** [comment]
All the `openai` tests fails. My guess is the `OPENAI_API_KEY` is no longer valid @ahuang11

File: mkdocs.yml [hunk 1]
```diff
@@ -59,9 +59,11 @@ watch:
 
 nav:
     - Home: index.md
+    - Awesome: awesome_list.md
```
Comment (ahuang11) [inline mkdocs.yml:62]: I'm not sure I prefer `Awesome` or something else like `External Resources`. Also, unsure if it should be the second item here, or the last item.

File: docs/awesome_list.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Links: [Medium](https://sophiamyang.medium.com/building-ai-chatbots-with-mistral-and-llama2-9c0f5abc296c)
+
+[![ChatBot with Mistral and Llama2](assets/thumbnails/mistral_and_llama.png)](https://sophiamyang.medium.com/building-ai-chatbots-with-mistral-and-llama2-9c0f5abc296c)
+
+## Ragna
+
+[Ragna](https://ragna.chat/) is an open source RAG orchestration framework. The UI is built on top of Panel.
+
+[<img style="height:315px;" src="../assets/images/ragna-logo.png"></img>](https://ragna.chat/)
```
Comment (ahuang11) [inline docs/awesome_list.md:95]: I think this is missing.

Comment (ahuang11) [reply docs/awesome_list.md:95]: I think `..` might not be needed?

image

File: docs/awesome_list.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### Chat with +1000 Python Libraries
+
+Build a chat bot on top of the Fleet AI embeddings.
+
+Authors: [Andrew Huang](https://twitter.com/IAteAnDrew1) | [Sophia Yang](https://twitter.com/sophiamyang)
+
+Link: [Blog Post](https://blog.holoviz.org/posts/fleet_ai/)
+
+[<img style="width:560px;height:315px" src="../assets/images/panel-chat-fleet-ai.jpg"></img>](https://blog.holoviz.org/posts/fleet_ai/)
```
Comment (ahuang11) [inline docs/awesome_list.md:67]: The `..` might not be needed here.

**Review by ahuang11 — COMMENTED** [review]
Thanks so much for adding these! I think these will be extremely valuable! I left some comments about broken links and thoughts on the category naming.

File: mkdocs.yml (hunk 1 above)
Comment (MarcSkovMadsen) [reply mkdocs.yml:62]: External Resources would also work for me.
Moving it to the end would also work for me.

Please let me know what I should change if any. Thanks.

**Comment by ahuang11** [comment]
Test seems unrelated. Merging. Thank you!

**Review by ahuang11 — APPROVED** [review]

**Pull request merged**